isthmus/src/main/java/io/substrait/isthmus/SubstraitRelNodeConverter.java

```diff
@@ -211,12 +211,15 @@ public RelNode visit(Set set) throws RuntimeException {
             input -> {
               relBuilder.push(input.accept(this));
             });
+    // MINUS_MULTISET and INTERSECTION_PRIMARY are set to be removed
+    // (substrait-io/substrait/pull/708)
```

Comment:
I don't think we're actually planning on removing those SetOps entirely from the core spec. What we do want to do is remove them here from the mapping to Calcite, because as described in the Substrait spec they don't map to SQL `EXCEPT` and `INTERSECT` which is what the Calcite relations do.

Reply:
Created an issue to track this as well
https://github.com/substrait-io/substrait-java/issues/303